lib/model/query/entities.js

```diff
@@ -132,7 +132,16 @@ createVersion.audit = (updatedEntity, dataset, partial, subDef) => (log) => {
 };
 createVersion.audit.withResult = true;
 
+////////////////////////////////////////////////////////////////////////////////
+// RESOLVE CONFLICT
 
+const resolveConflict = (entity, dataset) => ({ run }) => // eslint-disable-line no-unused-vars
+  run(sql`update entities set conflict=null where "id"=${entity.id}`);
+resolveConflict.audit = (entity, dataset) => (log) => log('entity.update.resolve', dataset, {
```

Comment:
If this new action is getting added, there are a couple other things to do
1. add it to the nonverbose and entity action catetory in model/query/audits
2. test that it comes through in the entity category filtering in the audit tests